src/layers/TrackerLayer.js

```diff
@@ -210,6 +210,16 @@ class TrackerLayer extends Layer {
     }
   }
 
+  /**
+   * Set the sort for tracker features.
+   * @param {Function} sort Sort function.
+   */
+  setSort(sort) {
+    if (this.tracker && sort) {
```

Comment:
```suggestion
    if (this.tracker) {
```
We want to be able to remove the sort if necessary